yvalenta/spec/spec_helper.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.